### ISSUE TITLE
有关每个 NToken 轨道可独立调节参数的提议

### ISSUE THREAD

**Issue by phantomk** [NEST-Protocol/nips#14]
鉴于每个 NToken 报价轨道的发展进度都不一致，需要每个轨道可独立调节参数（比如报价规模，手续费等）的功能，而不是所有 NToken 轨道都是一样的参数。
1. 例如对于轨道调用方来说，如果需要对该轨道有更高的报价密度的要求。就可以由调用方发起社区提案，独立调整该轨道的参数，而不会影响其他轨道。
2. 又例如已被调用的轨道可以先提高报价规模，以吸引更多的验证者，提高轨道精度。而不会影响小的轨道

**Comment by sunphen** [comment]
这个想法很好，我们不谋而合。每个预言机背后的token市值、交易量、社区基数不一样，差异化的预言机，应该设置差异化的参数。同时，我建议用QP token对预言机调整相关参数的社区提案进行投票，一方面降低nest dao的治理难度，另一方面提高QP token持有者参与治理的积极性。

**Comment by phantomk** [comment]
> 机
用对应 QP token 进行投票的想法也很好，增加了 QP token 的应用场景

**Comment by phantomk** [comment]
> 这个想法很好，我们不谋而合。每个预言机背后的token市值、交易量、社区基数不一样，差异化的预言机，应该设置差异化的参数。同时，我建议用QP token对预言机调整相关参数的社区提案进行投票，一方面降低nest dao的治理难度，另一方面提高QP token持有者参与治理的积极性。

用对应 QP token 进行投票的想法也很好，增加了 QP token 的应用场景。这样  QP token 就有了治理权和收益权

**Comment by JohnChanx** [comment]
> 这个想法很好，我们不谋而合。每个预言机背后的token市值、交易量、社区基数不一样，差异化的预言机，应该设置差异化的参数。同时，我建议用QP token对预言机调整相关参数的社区提案进行投票，一方面降低nest dao的治理难度，另一方面提高QP token持有者参与治理的积极性。

For me, Nest is more like the governance token of oracle, and ntoken is more like a kind of proof of stake. Hope nestcore will go further more on describing ideas of governance.

**Comment by sunphen** [comment]
> > 这个想法很好，我们不谋而合。每个预言机背后的token市值、交易量、社区基数不一样，差异化的预言机，应该设置差异化的参数。同时，我建议用QP token对预言机调整相关参数的社区提案进行投票，一方面降低nest dao的治理难度，另一方面提高QP token持有者参与治理的积极性。
> 
> For me, Nest is more like the governance token of oracle, and ntoken is more like a kind of proof of stake. Hope nestcore will go further more on describing ideas of governance.

Yes, QP token can participate in voting to adjust the parameters of Oracle orbit (such as the service charge of calling Oracle, quotation fee, quotation scale, etc.), so it has certain governance function.
